### ISSUE TITLE
CPU-only support

### ISSUE THREAD

**Issue by zplizzi** [zplizzi/tensorflow-fast-rcnn#6]
Copied from #2.

I got a problem when compiling the roi_pooling_op. Since I do not have GPU installed, I have configured bazel using cpu only. But when I tried to load the lib from python repl, it says:

```
tensorflow.python.framework.errors.NotFoundError: dlopen(/Users/dtong/code/data/tensorflow/bazel-bin/tensorflow/core/user_ops/roi_pooling_op_grad.so, 6): Symbol not found: __Z27ZeroTensorGpuKernelLauncherPfi
```

Do you have any idea of bazel configuration that could ignore the gpu code?

**Comment by zplizzi** [comment]
Unfortunately, I don't think CPU-only compilation will work with the code as-is. I tried to share some code between the two versions, and it looks like that's breaking CPU-only compatibility. If you need this, you can probably just slightly modify the code to remove the GPU parts entirely.  It'd probably be slightly more difficult to better separate the versions such that it can be compiled either way. I'd definitely welcome any contributions to that regard, though!

**Comment by tongda** [comment]
Yes, I understand. I have just commented lines related to GPU code. Thanks.
